src/main/java/com/kodilla/ecommercee/exception/GlobalHttpErrorHandler.java

```diff
@@ -0,0 +1,17 @@
+package com.kodilla.ecommercee.exception;
+
+import com.kodilla.ecommercee.domain.User;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class GlobalHttpErrorHandler {
```

Comment:
What is the point implementing this global handler? I think, the OrderNotFoundException works well with only annotation '@ResponseStatus'...?

Reply:
We would have all exceptions in one class so that would be more readable and also errors are more User friendly I think

Reply:
Ok, but if we (all team) accept this approach, implementing this in other controllers should be done after you merge this to master.

Reply:
I commited it to only showcase how I did it of course we all have to agree to one approach, and if we commit to use global  I think that  '@ResponseStatus' at exception class is not necessary as we throw it 'return ResponseEntity.status(HttpStatus.NOT_FOUND).body(exception.getMessage());'